modules/localgov_review_date/localgov_review_date.module

```diff
@@ -0,0 +1,65 @@
+<?php
+
+/**
+ * @file
+ * Localgov Review Date module hooks.
+ */
+
+use Drupal\Core\Entity\EntityTypeInterface;
+use Drupal\Core\Field\BaseFieldDefinition;
+use Drupal\Core\Form\FormStateInterface;
+
+/**
+ * Implements hook_entity_bundle_field_info().
+ */
+function localgov_review_date_entity_bundle_field_info(EntityTypeInterface $entity_type, $bundle, array $base_field_definitions) {
+
+  // Add the review status field to nodes configured for scheduled transitions.
+  if ($entity_type->id() === 'node') {
+
+    $scheduled_config = \Drupal::service('config.factory')->get('scheduled_transitions.settings');
+    if (is_null($scheduled_config)) {
+      return;
+    }
+
+    $scheduled_bundles = $scheduled_config->get('bundles');
```

Comment:
Seems that Scheduled Transitions doesn't quite work like this. It will store configuration for content type bundles that don't have enabled transitions.

Looking at how Scheduled Transitions does it on their config form https://git.drupalcode.org/project/scheduled_transitions/-/blob/1b3d8c51108ad7dbf87611e1fac14410592d9c2f/src/Form/ScheduledTransitionsSettingsForm.php#L140 they're using a utility service `scheduled_transitions.utility`

Reply:
I have update this to use `$scheduled_bundles = \Drupal::service('scheduled_transitions.utility')->getBundles();`